content/posts/hiero_links_of_august_third.md

```
@@ -0,0 +1,29 @@
+Below are links to some of the things that happened this past week. **Check it out!**
+
+---
+**[Angelina C](https://github.com/aceppaluni)** added [coderabbit planning](https://github.com/hiero-ledger/hiero-website/pull/540) to the Hiero Website to assist developers in working on issues.
+
+**[Phillip N](https://github.com/phillip-nyinomujuni)** migrated deprecated [set_amount_in_tinybars()](https://github.com/hiero-ledger/hiero-sdk-python/pull/2501) calls to set_hbar_amount(Hbar.from_tinybars(...)) in the Python SDK's TCK handler and consensus example, eliminating deprecation warnings while preserving behavior and keeping deprecated test coverage intact.
+
+**[Boris B](https://github.com/boris-bonin)** updated the [documentation](https://github.com/hiero-ledger/solo/pull/5613) for the Hiero Solo repository to maintain up-to-date information for the Solo CLI.
+
+**[Andrew B](https://github.com/andrewb1269)** updated the [pinned solo](https://github.com/hiero-ledger/hiero-sdk-java/pull/2868) version used in the CI across three jobs in the Java SDK. 
+
+**Want to be featured on the Hiero-Website?**
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)